imageio/plugins/spe.py

```diff
@@ -533,37 +382,70 @@ def extract_metadata(meta: Mapping, char_encoding: str = "latin1"):
         meta.pop("exposure_sec")
 
 
-class SpeFormat(Format):
-    """See :mod:`imageio.plugins.spe`"""
+class SpePlugin(PluginV3):
+    def __init__(
+        self,
+        request: Request,
+        check_filesize: bool = True,
+        char_encoding: Optional[str] = None,
+        sdt_meta: Optional[bool] = None,
+    ) -> None:
+        """Instantiate a new SPE file plugin object
 
-    def _can_read(self, request):
-        return request.extension in self.extensions
+        Parameters
+        ----------
+        request : Request
+            A request object representing the resource to be operated on.
+        check_filesize : bool
+            If True, compute the number of frames from the filesize, compare it
+            to the frame count in the file header, and raise a warning if the
+            counts don't match. (Certain software may create files with
+            incorrect counts in the header.)
```

Comment:
```suggestion
        char_encoding : str
            Deprecated. Exists for backwards compatibility; use ``char_encoding`` of
            ``metadata`` instead.
        sdt_control
            Deprecated. Exists for backwards compatibility; use ``sdt_control`` of
            ``metadata`` instead.
```

Same here, let's add a note to not use these parameters.